src/groupeddataframe/grouping.jl

```diff
@@ -344,6 +348,20 @@ julia> combine(:c => sum, gd)
 │ 3   │ 3     │ 10    │
 │ 4   │ 4     │ 12    │
 
+julia> combine(gd, [:b, :c] => sum)
```

Comment:
Can you show an example using `.=>` and mixing a single pair and a vector? I've removed that from the signature for simplicity.

Reply:
added.